### DISCUSSION TITLE
Report links

### DISCUSSION THREAD

**Discussion by GeorgeSG** [GeorgeSG/sub.rehab#9] [General]
Please use this thread as a temporary solution to report broken links or links that need correction. I'll try to keep up 🚀

If you want, you can also open PRs to fix broken / miscategorized links. Thank you! 🙏

**Comment by Evoludo** [comment]
Hi, I'm not at a computer at the moment so I can't open a PR, so can someone add a link for /r/androgyny to https://lemmy.blahaj.zone/c/androgyny for me please? This is the official new home that's listed on the subreddit. Cheers!

**Comment by noodlejetski** [comment]
/r/AnarchyChess shows two links that are basically the same:

1. https://sopuli.xyz/c/anarchychess
2. https://lemmy.world/c/anarchychess@sopuli.xyz

the second one is the same as the first one, just opened from lemmy.world instance.

also, for /r/AmITheAsshole, https://kbin.social/m/amitheasshole/microblog links to the microblog subsection of the magazine. main link is https://kbin.social/m/amitheasshole.

edit: the name of "/r/ich_iel" is mistyped as "ich_lel" (with an L instead of the I).

**Reply by GeorgeSG** [reply]
Resolved. Thanks! For the microblog - that's what was submitted. I'll leave it for now.

**Comment by ian-mitch** [comment]
Hello,

https://lemmy.world/c/ukcasual is a replacement for /r/casualuk not /r/ukcasual
Thanks!

**Comment by Pilirin** [comment]
hullo.

https://old.reddit.com/r/youtubedrama has moved to https://kbin.social/m/youtubedrama

https://old.reddit.com/r/perfectlycutmeows has places set up at https://kbin.social/m/perfectlycutmeows and at https://lemmy.world/c/perfectlycutmeows tho is still operating from the hellsite at the moment. not sure if that community will cross over, but it's set up for it anyway.

**Comment by GeorgeSG** [comment]
Hello folks. This thread is not meant for suggesting new links, but reporting issues with existing ones. You can suggest new links ~~once we reopen~~ from the form on the website. Thank you!

**Comment by noodlejetski** [comment]
https://lemmy.ml/c/linux_gaming@lemmy.ml could be just reformatted to https://lemmy.ml/c/linux_gaming for simplicity's sake. it currently uses a URL format for a community from another instance, but viewed from the same one, which doesn't really make much sense.

**Comment by ZeroPaladn** [comment]
<img width="286" alt="image" src="https://github.com/GeorgeSG/sub.rehab/assets/22324552/9c002d3c-817b-4784-8a1e-66b0fbd5c2b1">

The links lemmy.world/c/buildapc and squabbles.io/s/buildapc are not associated or related to r/buildapc in any way. We do not own these links nor are we associated with the owners. Please remove these links.

If you require verification, reach out to us through sub modmail here: https://www.reddit.com/message/compose?to=%2Fr%2Fbuildapc&subject=&message=

**Reply by GeorgeSG** [reply]
Hi @ZeroPaladn,

`lemmy.world/c/buildapc` and `squabbles.io/s/buildapc` are not marked as official (this is the "verified"-like checkmark icon on the right of `discord.gg/buildapc`). This clearly indicates they are not associated with `r/builadapc`. They would be filtered out if people filter by "Official".

I will not be taking the links down.

If you decide to create your own alternative on any of those platforms and mention it as an official alternative in `r/buildapc` I'd be happy to list it on sub.rehab and mark it as official.

---

On a personal, non-sub.rehab-official note, it is pretty concerning and uncool that you are trying to censor alternatives. As a `r/buildapc` lurker, I am disappointed.

**Reply by ZeroPaladn** [reply]
Censorship to alternatives was never the intention, only to not promote links that we aren't associated with. The site is laid out in a manner that promotes completely unrelated communities to r/buildapc and we aren't looking to mislead our users into thinking that we've jumped ship or have an alternate platform (when that time comes, we'll clearly announce it). We've already been approached by users asking if it's ours our not. Without knowing if they happened upon those communities from your links, we're not sure if your ✅ is satisfactory to ensuring that r/buildapc isn't misrepresented.

**Reply by GeorgeSG** [reply]
@ZeroPaladn, glad to have that cleared up and apologies that my initial response was a bit too accusatory!

The goal of the checkmark is to clearly indicate exactly which alternatives are officially endorsed by the original subreddit and which are community spin-offs. 

We are trying to indicate and promote official alternatives by:

1. Marking links as official only if they have been endorsed officially in the original subreddit (sidebar, pinned post, subreddit privated message). We have a moderation process setup to verify that.
3. Showing official links at the top of the list for each subreddit
4. Showing a tooltip when you hover the checkmark (screenshot 1)
5. Allowing users to filter only official links and showing a tooltip when you hover the "official" filter (screenshot 2) - i think the wording on that one can be improved. Maybe "Communities that have been endorsed or created by the original subreddit staff"?
6. A section in the FAQ page that explains what "official" is (screenshot 3)


I am open to suggestions on how to improve the wording and visual clarity of the indicators. I think having a bigger visual gap between official and unofficial links is possible and viable.

I am not a native speaker, so I could definitely use the help on wording as well 😅

The goal of sub.rehab however is to list alternatives - not only officially endorsed ones. If people decide to stop using reddit and want to use other platforms, I want to help them find communities they are interested in. So I will not delist unofficial links - I'd rather shut down the entire site than do that :) 

---

Checkmark hover:
![image](https://github.com/GeorgeSG/sub.rehab/assets/2514425/a86667d0-391a-4838-91f5-8a524d4f8c3f)

Filter hover:
![image](https://github.com/GeorgeSG/sub.rehab/assets/2514425/2a69c09e-ce41-4eb9-a835-c0dac636d2ea)

FAQ section:
![image](https://github.com/GeorgeSG/sub.rehab/assets/2514425/f4b5196c-7fef-490d-81b3-482d5ad82873)

**Reply by GeorgeSG** [reply]
Having thought about this a bit more, I made the following changes:

1. Improved the "official" filter explanation:
<img width="610" alt="image" src="https://github.com/GeorgeSG/sub.rehab/assets/2514425/25ba960f-0499-4443-9d23-7ff869ec9f27">

2. Improved the FAQ sections of the website:
<img width="444" alt="image" src="https://github.com/GeorgeSG/sub.rehab/assets/2514425/7903f867-45ab-4067-ac8b-9cfba38de0e1">
<img width="449" alt="image" src="https://github.com/GeorgeSG/sub.rehab/assets/2514425/9aa37897-e88d-49d1-849c-eb0fab8a14a2">

3. Added more distinct visual separation between official links and spin-offs:
<img width="369" alt="image" src="https://github.com/GeorgeSG/sub.rehab/assets/2514425/6f514730-0dd8-4ef3-9d90-bcbb43e881ce">


I hope this alleviates the concerns of misrepresentation and resolves this thread.

**Reply by ZeroPaladn** [reply]
Hey @GeorgeSG - appreciate the changes you've made, you've already done a lot of what I was going to present for edits and we're very satisfied with the new distinction and explanations added. 

Appreciate the continued effort here - consider this thread resolved!

**Comment by dojoteef** [comment]
Please change r/machinelearning to official. See this stickied post:
https://old.reddit.com/r/MachineLearning/comments/14ionyi/additional_resources/

**Reply by GeorgeSG** [reply]
Thanks, updated! And thanks for linking sub.rehab in your pinned post!

**Comment by NicoCharrua** [comment]
Under 196, the kbin alternative is listed as https://fedi196.gay/, which is the instance, not the magazine (community). I think the link should be changed to https://fedi196.gay/m/196, since that links to the magazine, and it would work properly with the home instance feature.

Worth noting that the mods of r/196 are the ones who linked to the whole instance instead of the magazine in [their announcement](https://www.reddit.com/r/196/comments/146y2qw/r196_offsite_migration_discord_info/), but I'm guessing this was a mistake since it just makes it more confusing for new users.

**Reply by GeorgeSG** [reply]
Agreed. I think we will start to require a community specified for all lemmy links so the Home Instance feature can work.

**Comment by Thebestpercy** [comment]
When submitting a link I misspelled r/lotrmemes to r/lotrmems if that could be adjusted it would be greatly appreciated

**Reply by GeorgeSG** [reply]
Will fix, thank you!

**Comment by nyxiereal** [comment]
Broken link: https://ramble.pw/f/piracy

**Comment by ornato-t** [comment]
https://forum.basedcount.com/c/pcm is incorrectly labeled as a "spin off" and not official. I had already mentioned that we were official in the application form but that must've been overlooked. The community is being moderated by the original r/PCM mod team and is maintaned and hosted by the PCM bot team. 

Please let me know if further verification is required (some people might miss their notification and require a ping on Discord, with which I'll happily help).

![immagine](https://github.com/GeorgeSG/sub.rehab/assets/53061271/5572cdbc-f5e6-476e-8427-508a2b9dbd01)

**Reply by GeorgeSG** [reply]
We probably marked it as a spin-off, because there is no link/endorsement r/PoliticalCompassMemes. We typically look for a pinned post (or at least a post from one of the mods), sidebar link, or a link in the privated message (if subreddit was private).

I reached out via mod mail to verify.

**Reply by GeorgeSG** [reply]
Promoted to official.

**Reply by ornato-t** [reply]
Thanks!

**Reply by ornato-t** [reply]
Hey it's me again, I have one more issue.

We recently had some severe problems with our instance, which caused us to have to move domains, from `forum.basedcount.com` to `lemmy.basedcount.com`. As such PCM is now on https://lemmy.basedcount.com/c/pcm.

Could you please update the link on sub.rehab?

**Comment by muznyo** [comment]
The r/protonmail spinoff is [official](https://lemmy.world/comment/1204904), also it should reference r/protondrive, r/protonvpn and r/protonpass too, since its general community for proton's products

**Reply by GeorgeSG** [reply]
Unfortunately, this does not meet our criteria for marking something as official. It needs to be communicated in the original subreddit somehow, or on some other official channel.

**Reply by muznyo** [reply]
I don't know if this official enough, [but](https://lemmy.world/comment/1416169)

**Reply by muznyo** [reply]
Ok, so it's official, because proton mail account replied too

**Reply by muznyo** [reply]
Just a reminder you haven't updated it yet

**Comment by NicoCharrua** [comment]
Under r/braves, https://fanaticus.social/c/braves is marked as misc when it should be lemmy

**Comment by NicoCharrua** [comment]
Broken link for discord under r/weirdal link. It's https://4wktfg2/ , should be https://discord.gg/4wKTfG2

**Reply by GeorgeSG** [reply]
Thanks, fixed!

**Comment by MaillPierre** [comment]
Broken link for r/RimWorld, sends to https://rimworldporn.cloud/m/RimWorldPorn but the page redirects to a 404 error.

**Comment by NicoCharrua** [comment]
https://lemmy.world/c/guiltygear should be marked as official.

https://old.reddit.com/r/Guiltygear/comments/14fv0t1/rguiltygear_on_lemmy/

**Comment by kitten7280** [comment]
r/FREEMEDIAHECKYEAH link to lemmy is broken. I guess they switched top-level domain.

The correct link is: https://lemmy.fmhy.net/c/freemediaheckyeah
Proof: https://www.reddit.com/r/FREEMEDIAHECKYEAH/comments/15l6tpe/we_fixed_lemmy/
Another proof: https://www.reddit.com/r/FREEMEDIAHECKYEAH/ (sidebar)
